starter-code/iteration-2/main.ts

```diff
@@ -1,6 +1,25 @@
 // Add the reference to the "TodoInterface"
+import {TodoInterface} from './todoInterface';
 
 // 1. Create a class Todo that implements the Interface created before.
+class Todo implements TodoInterface {
+    tasks: Array<string> = [];
+
+    addTask(task: string):void {
+        this.tasks.push(task)
+    }
+
+    listAllTasks():void {
+        this.tasks.forEach((task: string)=>{
+            console.log(task);
+        });
+    }
+
+    deleteTask(chosenTask) {
```

Comment:
this function should have a type declared:
`deleteTask(task: string): number {
    var index = this.todoList.indexOf(task);
    this.todoList.splice(index, 1);
    console.log(`Task "${task}" removed from the list`);
    return this.todoList.length;
  }
}`